src/components/TableResize.js

```diff
@@ -94,21 +94,32 @@ class TableResize extends React.Component {
   };
 
   onResizeStart = (id, e) => {
-    this.setState({ isResize: true, id, startPosition: e.clientX });
+    this.setState({ isResize: true, id });
   };
 
   onResizeMove = (id, e) => {
-    const { startPosition, isResize, resizeCoords } = this.state;
-
+    const { isResize, resizeCoords } = this.state;
+    const fixedMinWidth = 100;
+    const idNumber = parseInt(id);
+    const finalCells = Object.entries(this.cellsRef);
+    if (idNumber >= finalCells.length) return;
     if (isResize) {
-      const leftPos = startPosition - (startPosition - e.clientX);
+      let leftPos = e.clientX;
+      if (leftPos > resizeCoords[idNumber + 1].left - fixedMinWidth)
+        leftPos = resizeCoords[idNumber + 1].left - fixedMinWidth;
+      if (
+        (idNumber === 0 && typeof resizeCoords[0] != 'undefined') ||
+        (idNumber === 1 && typeof resizeCoords[0] == 'undefined')
```

Comment:
resizeCoords[0] is undefined when you set `selectableRows: "none"`
=== - changed this.

Reply:
Thanks for explaining. That's definitely something that is going to be difficult for me to remember if I need to look at this again in a few months. I think it would be better if we could code it in such a way that it documents itself a little more. Perhaps by actually using the `selectableRows` options, since this is the situation we care about?

Reply:
at present moment options and selectableRows is not passed to TableResize as props. Added comment explaining this line. Let me know if this is sufficient, or if I should pass props for selectableRows in MUIDataTable.js

Reply:
Forwarded all options props that are passed to MUIDataTable